package.json

```diff
@@ -32,14 +32,16 @@
     "test": "jest"
   },
   "dependencies": {
-    "forta-agent": "^0.1.45"
+    "forta-agent": "^0.1.45",
+    "forta-agent-tools": "^3.2.10"
```

Comment:
Since your bot wouldn't be using it in production, `forta-agent-tools` could be moved to `devDependencies`.